turbo.json

```diff
@@ -1,75 +1,131 @@
 {
-	"$schema": "https://turborepo.org/schema.json",
-	"pipeline": {
-		"@sveltejs/kit#build": {
-			"dependsOn": ["^build"],
-			"inputs": ["rollup.config.js", "src/**", "scripts/**", "types/**"],
-			"outputs": ["dist/**", "docs/**", "assets/**"],
-			"outputMode": "new-only"
-		},
-		"kit.svelte.dev#build": {
-			"dependsOn": ["^build", "$VERCEL", "$ENABLE_VC_BUILD"],
-			"inputs": ["src/**", "../../packages/kit/docs/**", "../../documentation/**"],
-			"outputs": [".vercel_build_output/**", ".vercel/output/**"],
-			"outputMode": "new-only"
-		},
-		"build": {
-			"dependsOn": ["^build", "$VERCEL", "$ENABLE_VC_BUILD"],
-			"inputs": ["src/**", "scripts/**", "shared/**", "templates/**"],
-			"outputs": [
-				"files/**",
-				"dist/**",
-				".svelte-kit/**",
-				".vercel_build_output/**",
-				".vercel/output/**"
-			],
-			"outputMode": "new-only"
-		},
-		"check": {
-			"dependsOn": ["prepare"],
-			"inputs": [
-				"src/**",
-				"types/**",
-				".svelte-kit/types/**",
-				".custom-out-dir/types/**",
-				".svelte-kit/tsconfig.json"
-			],
-			"outputs": [],
-			"outputMode": "new-only"
-		},
-		"format": {
-			"cache": false,
-			"outputs": [],
-			"outputMode": "new-only"
-		},
-		"prepare": {
-			"inputs": ["src/**", "source/**"],
-			"outputs": [
-				".svelte-kit/types/**",
-				".svelte-kit/tsconfig.json",
-				".custom-out-dir/types/**",
-				".custom-out-dir/tsconfig.json"
-			],
-			"outputMode": "new-only"
-		},
-		"test": {
-			"dependsOn": ["^build", "$CI", "$TURBO_CACHE_KEY"],
-			"outputs": ["coverage/", "test-results/**"],
-			"outputMode": "new-only"
-		},
-		"lint": {
-			"outputs": [],
-			"outputMode": "new-only"
-		},
-		"precommit": {
-			"dependsOn": ["format", "lint"],
-			"outputs": [],
-			"outputMode": "new-only"
-		},
-		"dev": {
-			"cache": false,
-			"outputMode": "new-only"
-		}
-	},
-	"globalDependencies": [".eslintrc.json", ".prettierrc", "tsconfig.json"]
+  "$schema": "https://turborepo.org/schema.json",
```

Comment:
accidental formatting (now has spaces instead of tabs)? Anything that actually changed in here?

Reply:
this was done by turbos migration tool and format didn't bother so I kept it that way. The code change is the way env dependencies are declared

Reply:
i just checked what happens and it looks like prettier is never run from the root of the monorepo, https://github.com/sveltejs/kit/blob/edd815ed7bba7ab88461f9291f924c05a01d1328/package.json#L11  this recursively calls format scripts in packages, but there is no separate format script in root so prettier is never called on root files.

i have formatted it manually with `pnpm prettier --write turbo.json` to keep the diff smaller here, but we should look into fixing this in a separate PR